### PR TITLE
[CLEANUP] Stop using Translator and TranslatorRegistry

### LINKED ISSUE #702: Stop using Translator and TranslatorRegistry
Instead, use the TYPO3 Core or Extbase localization functions.

### PR DESCRIPTION
**Pull request by Konafets**
Closes #702

### CODE REVIEW

File: Classes/ViewHelper/Countdown.php [hunk 1]
```diff
@@ -137,6 +123,13 @@ protected function getAsSeconds(int $seconds): string
      */
     protected function getFormattedMessage(int $countdownValue, string $countdownText): string
     {
-        return \sprintf($this->translator->translate('message_countdown'), $countdownValue, $countdownText);
+        return \sprintf(
+            LocalizationUtility::translate(
```
Comment (oliverklee) [inline Classes/ViewHelper/Countdown.php:127]: Let let's put this `translate` call into a single line to make this method less long (vertically).

File: Classes/Email/Salutation.php [hunk 2]
```diff
@@ -109,15 +95,20 @@ public function createIntroduction(string $introductionBegin, \Tx_Seminars_OldMo
         if (!$event->hasDate()) {
             return $result;
         }
-
-        $result .= ' ' . sprintf($this->translator->translate('email_eventDate'), $event->getDate('-'));
+        $result .= ' ' . sprintf(
+            LocalizationUtility::translate(
```
Comment (oliverklee) [inline Classes/Email/Salutation.php:99]: Let let's put this `translate` call into a single line to make this method less long (vertically).

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Comment by Konafets** [comment]
Wondering, why suddenly an unrelated test fails. Will investigate this tomorrow.

**Comment by oliverklee** [comment]
The failure probably is an incomplete mock: https://github.com/oliverklee/ext-seminars/issues/718

**Review by oliverklee — APPROVED** [review]

**Pull request merged**